src/Mod/Part/App/TopoShapeExpansion.cpp

```diff
@@ -4157,14 +4157,23 @@ TopoShape& TopoShape::makeElementLoft(const std::vector<TopoShape>& shapes,
                   "Need at least two vertices, edges or wires to create loft face");
     }
 
+    int i=0;
+    Base::Vector3d center1,center2;
     for (auto& sh : profiles) {
+        if (i>0) {
+            if (sh.getCenterOfGravity(center1) and profiles[i-1].getCenterOfGravity(center2) and center1.IsEqual(center2,Precision::Confusion())) {
```

Comment:
While it's technically not wrong to use `and` in c++, I did not find any other instances of it in FreeCAD, so maybe we should go with the more common `&&` operator?
```suggestion
            if (sh.getCenterOfGravity(center1) && profiles[i-1].getCenterOfGravity(center2) && center1.IsEqual(center2,Precision::Confusion())) {
```

Reply:
i changed it to &&